# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/cocoa-foundation/Cargo.toml
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/cocoa-foundation/Cargo.toml b/third_party/rust/cocoa-foundation/Cargo.toml
deleted file mode 100644
--- a/third_party/rust/cocoa-foundation/Cargo.toml
+++ /dev/null
@@ -1,42 +0,0 @@
-# THIS FILE IS AUTOMATICALLY GENERATED BY CARGO
-#
-# When uploading crates to the registry Cargo will automatically
-# "normalize" Cargo.toml files for maximal compatibility
-# with all versions of Cargo and also rewrite `path` dependencies
-# to registry (e.g., crates.io) dependencies
-#
-# If you believe there's an error in this file please file an
-# issue against the rust-lang/cargo repository. If you're
-# editing this file be aware that the upstream Cargo.toml
-# will likely look very different (and much more reasonable)
-
-[package]
-name = "cocoa-foundation"
-version = "0.1.0"
-authors = ["The Servo Project Developers"]
-description = "Bindings to Cocoa Foundation for macOS"
-homepage = "https://github.com/servo/core-foundation-rs"
-license = "MIT / Apache-2.0"
-repository = "https://github.com/servo/core-foundation-rs"
-[package.metadata.docs.rs]
-default-target = "x86_64-apple-darwin"
-[dependencies.bitflags]
-version = "1.0"
-
-[dependencies.block]
-version = "0.1"
-
-[dependencies.core-foundation]
-version = "0.9"
-
-[dependencies.core-graphics-types]
-version = "0.1"
-
-[dependencies.foreign-types]
-version = "0.3"
-
-[dependencies.libc]
-version = "0.2"
-
-[dependencies.objc]
-version = "0.2.3"